stress_test/class_test/emulator_test_mtcbench.json

```diff
@@ -0,0 +1,25 @@
+{
+
+"SB-Emulator_name":"MTCBench",
```

Comment:
to be consistent with any other JSON configuration key, follow this naming scheme:
- no capital letters
- use underscores 

e.g., `sbemu_node_ip`